packages/cli-repl/src/arg-parser.ts

```diff
@@ -106,9 +106,7 @@ const DEPRECATED_ARGS_WITH_REPLACEMENT: Record<string, string> = {
 const UNSUPPORTED_ARGS: Readonly<string[]> = [
   'eval',
   'sslFIPSMode',
-  'tlsFIPSMode',
-  'sslCertificateSelector',
```

Comment:
We should probably update the help text of the CLI to include the flags and also add a comment in there to say that it's currently only supported on Windows.